WikiDrips/WikiTableViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  WikiTableViewController.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/1/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import UIKit
+
+class WikiTableViewController: UITableViewController {
+
+    var wikiDocs = [[WikiDoc]]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+//      TODO: replace with async call to Wikipedia API
+        self.wikiDocs.insert([WikiDoc()], at: 0)
+        self.wikiDocs.insert([WikiDoc()], at: 0)
+        self.wikiDocs.insert([WikiDoc()], at: 0)
+
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return wikiDocs.count
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return wikiDocs[section].count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "WikiDoc", for: indexPath) as! WikiTableViewCell
+        let wikiDoc = wikiDocs[indexPath.section][indexPath.row]
+        cell.wikiTitleLabel?.text = wikiDoc.title
+        cell.wikiDateLabel?.text = wikiDoc.editedDate()
```

Comment:
See my other comment about `DateFormatter`. Creating a DateFormatter for every cell will absolutely kill your scrolling performance.